README.md

```diff
@@ -1,23 +1,34 @@
-# SightGlass
+#  data-store-monitor
 
-SightGlass is a monitoring plugin for the [DataStore](https://github.com/HumanCellAtlas/data-store)
+This is a monitoring plugin for the [DataStore](https://github.com/HumanCellAtlas/data-store)
 
 ## DSS-Authentication:
 Due to the DSS requiring authentication for the /Subscription endpoint we must create a service account with GCP
 that will be utilized for the hca cli tool login.
 The service account is created for you, see [infra/google-account](infra/google-account) for more information
 Once the service account is created head over to [gcp-dashboard](https://console.developers.google.com/) and get the
-`google_application_credentials.json` associated with the account created. 
+`gcp_credentials.json` associated with the account created. 
 
 ```
 gcloud iam service-accounts keys create ${DSS_MON_HOME}/deployments/${DSS_INFRA_TAG_STAGE}/gcp-credentials.json --iam-account ${DSS_MON_GCP_SERVICE_ACCOUNT_NAME}@${GCP_PROJECT_NAME}.iam.gserviceaccount.com
 ```
 
+## Stages:
+Stage configuration files are stored in the (deployments)[deployments] directory.
+This includes a config file for the HCA tool, as well as the location to save the `gcp_credentials.json` for the applicable
+service account. 
+To work a different stage, perform:
+``` export DEPLOYMENT={STAGE} && source environment```
+
 ## Secrets:
-### HMAC:
-In order to ensure authenticity of notifications, and hmac key is used during the subscription process.
-Set a hmac key for as follows:
-	`openssl rand -base64 12 | scripts/set_secret.py --secret-name $DSS_MON_HMAC_SECRET_NAME`
+
 ### Webhook
-	The DSS-Monitor can provide notifications for daily progress in a slack channel. Add a webhook url as follows
-	`echo 'URL' |scripts/set_secret.py --secret-name $DSS_MONITOR_WEBHOOK_SECRET_NAME` 
\ No newline at end of file
+The DSS-Monitor can provide notifications for daily progress in a slack channel. Add a webhook url as follows
+	`echo 'URL' | scripts/set_secret.py --secret-name $DSS_MONITOR_WEBHOOK_SECRET_NAME` 
+	
+## Deployments:
+
+Once the appropriate service account config file has been saved according the instructions above, and there has been a
+webhook placed into the secrete manager, perform:
+`make DEPLOYMENT={STAGE} deploy`
```

Comment:
An operator would be able to perform the `export DEPLOYMENT={stage} && source environment` , followed by a `make deploy` which would source the environment file again during the make command, and run the deployment.
Alternatively, if the operator runs `make DEPLOYMENT={stage} deploy` the correct ENV would be sources, and then the deployment would run. It would just be an easier way to perform a similar operation. 

The `DEPLOYMENT` and `DSS_INFRA_TAG_STAGE` would evaluate to the same variable, they should not differ.

Reply:
^^^ that comment is no longer important, i made the deployment design way easier, 1 lambda to rule them all.